### PR TITLE
readme code updates and minor bugfixes

### PR DESCRIPTION
**Pull request by nagkumar91**
# Description

Updates to README.
Added `@experimental` to simulator and its call method.
Added the prompty being used in the simulator as a package resource to resolve import issues from notebooks.

# All SDK Contribution checklist:
- [x] **The pull request does not introduce [breaking changes]**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md).**

## General Guidelines and Best Practices
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md##building-and-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

**Review by needuv — APPROVED** [review]

File: sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/simulator/simulator.py [hunk 1]
```diff
@@ -57,6 +60,7 @@ def _validate_project_config(azure_ai_project: Dict[str, Any]):
             raise ValueError("subscription_id, resource_group_name, and project_name must not be None")
 
     # @monitor_task_simulator
+    @experimental
```
Comment (needuv) [inline sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/simulator/simulator.py:63]: if the class is already marked as experimental, you don't need to mark the methods on it as well. It's assumed everything inside is subject to change. Customers sometimes get confused when more than one thing is logged for the same class they are using

File: sdk/evaluation/azure-ai-evaluation/setup.py [hunk 2]
```diff
@@ -86,5 +86,6 @@
     },
     package_data={
         "pytyped": ["py.typed"],
+        "azure.ai.evaluation.simulator._prompty": ["*.prompty"],
```
Comment (needuv) [inline sdk/evaluation/azure-ai-evaluation/setup.py:89]: I recommend adding this to `MANIFEST.in` as the evaluation prompty files are included there

File: sdk/evaluation/azure-ai-evaluation/README.md [hunk 3]
```diff
@@ -57,9 +57,9 @@ if __name__ == "__main__":
 
     # Initialize Project Scope
     azure_ai_project = {
-        "subscription_id": "e0fd569c-e34a-4249-8c24-e8d723c7f054",
-        "resource_group_name": "rg-test",
-        "project_name": "project-test",
+        "subscription_id": <subscription_id>,
+        "resource_group_name": <resource_group_name>,
+        "project_name": <project_name>
```
Comment (needuv) [inline sdk/evaluation/azure-ai-evaluation/README.md:62]: Thanks for the clean up!

**Review by ninghu — APPROVED** [review]

File: sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/simulator/simulator.py (hunk 1 above)
Comment (nagkumar91) [reply sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/simulator/simulator.py]: Removed :)

**Pull request merged**